source/ada/lsp-lal_utils.ads

```diff
@@ -41,6 +44,11 @@ package LSP.Lal_Utils is
    --  Imprecise is set to True if LAL's imprecise fallback mechanism has been
    --  used to compute the cross reference.
 
+   function Get_Last_Name
```

Comment:
(style) join this line with the next line (same comment in the body)

Reply:
done